docs/dependencies.md

```diff
@@ -0,0 +1,213 @@
+# Dependencies
+
+## Core Spring Boot Dependencies
+
+- **spring-boot-starter-web**:  
+  Provides the essential components for building web applications with Spring MVC.
+
+- **spring-boot-starter-data-jpa**:  
+  Simplifies the integration of JPA (Java Persistence API) into Spring applications,
+  making it easier to work with relational databases.
+
+- **spring-boot-starter-data-rest**:  
+  Enables the creation of RESTful APIs from Spring Data repositories, allowing for easy
+  exposure of data as REST endpoints.
```

Comment:
It looks like the dependency also handles pagination and so on, so the description is a bit misleading here.